### DISCUSSION TITLE
Color chart area background only

### DISCUSSION THREAD

**Discussion by dgwyer** [observablehq/plot#973] [Q&A] [answered]
The docs show an example of how to customize plot styles.

https://observablehq.com/@observablehq/plot-cheatsheets-layouts#cell-60

In the example the _whole_ chart background is customized but I'd also like a way to color the chart area only (i.e. area inside the axes). Is this easily doable? If so what is the recommended approach?

**Comment by Fil** [comment, marked as answer]
You could start with `Plot.frame({fill: "red"})`
